sefaria/helper/topic.py

```diff
@@ -1261,4 +1261,23 @@ def delete_ref_topic_link(tref, to_topic, link_type, lang):
             link.delete()
             return {"status": "ok"}
         else:
-            return {"error": f"Cannot delete link between {tref} and {to_topic}."}
\ No newline at end of file
+            return {"error": f"Cannot delete link between {tref} and {to_topic}."}
+
+
+def add_image_to_topic(topic_slug, image_uri, en_caption, he_caption):
+    """
+    A function to add an image to a Topic in the database. Helper for data migration.
+    This function queries the desired Topic, adds the image data, and then saves.
+    :param topic_slug String: A valid slug for a Topic
+    :param image_uri String: The URI of the image stored in the GCP images bucket, in the topics subdirectory.
+                             NOTE: Incorrectly stored, or external images, will not pass validation for save
+    :param en_caption String: The English caption for a Topic image
+    :param he_caption String: The Hebrew caption for a Topic image
+    """
+    topic = Topic().load({'slug': topic_slug})
```

Comment:
Clearer to use `Topic.init(topic_slug)`. `.init()` is a function that applies to any object with a slug (these objects all inherit from `SluggedAbstractMongoRecord`)

Reply:
Okay, sounds good! Thank you!